libqtile/window.py

```diff
@@ -1307,6 +1314,45 @@ def cmd_resize_floating(self, dw, dh):
         """Add dw and dh to size of window"""
         self.tweak_float(dw=dw, dh=dh)
 
+    def cmd_start_resize(self):
+        start_x = None
+        start_y = None
+        x, y = self.qtile.mouse_position
+        if x < self.x + self.width // 2:
+            start_x = self.x
+        if y < self.y + self.height // 2:
+            start_y = self.y
+        self._resize_start = (self.width, self.height, start_x, start_y)
+        return (0, 0)
```

Comment:
Returning a fake value while putting the real stuff in the window object sounds nasty. Can't we respect the current `Drag` API or extend it?